documentation/models.py

```diff
@@ -294,15 +294,26 @@ def get_embed_app(self):
 
 
 class Map(Page, RichText, CloneableMixin):
+    blocks = models.ManyToManyField(Block, blank=True)
 
     def __unicode__(self):
         return self.title
 
     def get_absolute_url(self):
-        return '/%s/%s/' % (self.parent.slug, self.slug)
+        url = '/concepts/'
+
+        parent = self.parent
+
+        while parent is not None:
+            url += '%s/' % parent.slug
+            parent = parent.parent
+
+        url += '%s/' % self.slug
```

Comment:
could this just be something like `"/concepts/%s/%s" % (parent.get_absolute_url(), self.slug)`?

Reply:
oh, ofc! In that case this looks good